mmv1/third_party/terraform/website/docs/guides/version_6_upgrade.html.markdown

```diff
@@ -88,15 +88,6 @@ terraform {
 
 ## Provider
 
-### Compute: `name_prefix` max length has been extended from 37 to 54 characters
-
-Affected resources: `google_compute_instance_template`, `google_compute_region_instance_template`, `google_compute_ssl_certificate`,
-and `google_compute_region_ssl_certificate`
-
-Previously, the max length of `name_prefix` was 37 characters since the autogenerated UUID suffix was 26 characters which combined to
-the total max length for names of 63 characters.
-In 6.0, providing a `name_prefix` larger than 37 characters will prompt the provider to use a shortened suffix of only 9 characters, leading to a new max of 54 characters for `name_prefix`. This shortened suffix is inevitably more prone to collisions, so use the longer max `name_prefix` length with caution.
-
 ### Provider attribution label is enabled by default
```

Comment:
```suggestion
## Provider: Terraform provider attribution label is added to new resources by default
```

Instead of a provider H2 and subitem H3s, can we make the H2s prefixed with `Provider:`? (same with a Removals section if we had one) That generates a better TOC where some of the highest impact changes in older releases are the hardest to scan for:

<img width="449" alt="image" src="https://github.com/user-attachments/assets/55557502-eb06-4b31-9a0e-9893e66fc500">

Reply:
(also minor clarification in the title about affecting new resources. Feel free to break up the change if you disagree with either!)

Reply:
updated